Add LOGO XD

Nir-betesh opened the pull request:
Add logo and some logic

almogkh commented:
You should make sure you're using JavaFX runtime version 8 because that's the JRE version we're using. Notice you've changed the version in the FXML files from 8 to 18.

Nir-betesh commented:
I added the line in the file that you edit on Ofek’s computer, also i have
changed the 18 to 8 in FXMLs files but it keep returning to 18 🤷🏽‍♂️

On Mon, 19 Dec 2022 at 2:56 Almog Khaikin ***@***.***> wrote:

> You should make sure you're using JavaFX runtime version 8 because that's
> the JRE version we're using. Notice you've changed the version in the FXML
> files from 8 to 18.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/almogkh/Ekrut-Project/pull/17#issuecomment-1356936892>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A3Y24WLYYTSSNQHSU7E2NGTWN6XEXANCNFSM6AAAAAATCYZM34>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

almogkh commented on EKrut-Client/src/gui/ClientLoginController.java:
```diff
@@ -10,6 +10,9 @@
 import javafx.scene.control.TextField;
 import javafx.scene.paint.Color;
 import javafx.stage.WindowEvent;
+import javax.swing.JButton;
```
This does not need to be here.  Please delete this line.

almogkh commented on EKrut-Client/src/gui/ClientLoginController.java:
```diff
@@ -68,14 +70,16 @@ void attemptLogin(ActionEvent event) throws IOException {
+	
+	// Press enter from anywhere in scene in order to logged in.
+	public void enterKeyForLogin() {
+		loginBtn.setDefaultButton(true);
```
If you set the default button property of the button in the FXML, this is unnecessary since you never change the default to anything else.
